README.md

```diff
@@ -240,7 +240,7 @@ The contents of the User-Agent header of the request.
 
 ### morgan.compile(format)
 
-Compile a format string into a `format` function for use by `morgan`. A format string
+Compiles a format string into a `format` function for use by `morgan`. A format string
```

Comment:
This isn't a typo. If you still believe so, please elaborate.

Reply:
I will undo it